vector/src/main/java/im/vector/app/features/roomdirectory/createroom/CreateRoomActivity.kt

```diff
@@ -53,7 +53,10 @@ class CreateRoomActivity : VectorBaseActivity<ActivitySimpleBinding>(), ToolbarC
             addFragment(
                     R.id.simpleFragmentContainer,
                     CreateRoomFragment::class.java,
-                    CreateRoomArgs(intent?.getStringExtra(INITIAL_NAME) ?: "")
+                    CreateRoomArgs(
+                            intent?.getStringExtra(INITIAL_NAME) ?: "",
+                            isSubSpace = intent?.getBooleanExtra(IS_SPACE, false) ?: false
```

Comment:
Why there is a naming diff between `isSubSpace` and `IS_SPACE` ? Rename the latter to `IS_SUB_SPACE`?

Reply:
renamed to isSpace